tests/browser/test/extensive_test.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Node.js script to run Automated tests in Chrome, via webdriver.
+ */
+
+const chai = require('chai');
+const {
+  testSetup,
+  testFileLocations,
+  getBlockTypeFromWorkspace,
+} = require('./test_setup');
+const {Key} = require('webdriverio');
+
+let browser;
+suite(
+  'Testing Blockly Loading Large Configurations and Deletion',
```

Comment:
Can you change your suite name to "Loading Large Configuration and Deletion"?

Generally for test suites I like to follow the convention of "This tests X" where X becomes the suite title!